packages/appcd-core/CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
-# v1.1.4
+# v2.0.0
```

Comment:
Missing the version bump in package.json? I guess we'd also need to update the plugins to support this version too?